include/rdma/fi_log.h

```diff
@@ -35,8 +35,6 @@
 #ifndef FI_LOG_H
 #define FI_LOG_H
 
-#include "config.h"
```

Comment:
This needs to be included here, `ENABLE_DEBUG` is used lower in the file. This will break logging.